AppCenter/AppCenter/MSAppCenter.m

```diff
@@ -285,6 +280,13 @@ - (BOOL)startService:(Class)clazz andSendLog:(BOOL)sendLog {
 
     // Start service with log manager.
     [service startWithLogManager:self.logManager appSecret:self.appSecret];
+    
+    // Disable service if AppCenter is disabled.
+    if ([clazz isEnabled] && !self.isEnabled) {
+      self.enabledStateUpdating = YES;
```

Comment:
@TroubleMakerBen @clpolet Do we really need this flag? I wonder if we synchronize `setEnabled` and get rid of this flag.

Reply:
@MatkovIvan Do you need to change this flag? You are not disabling the core services here.

Reply:
@jaelim-ms This is caused by https://github.com/Microsoft/AppCenter-SDK-Apple/blob/9449f274db22f9d5c3777a3c749b4b1ff0afdf53/AppCenter/AppCenter/MSServiceAbstract.m#L88

Reply:
Gotcha